site/content/tutorial/02-reactivity/05-reactivity-and-functions/text.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Reactivity and Functions
+---
+
+Svelte reactivity can be slightly unintuitive once you start declaring your own functions.
+
+For example, you might have some code where a reactive variable is becoming a bit complex:
+
+```js
+let x = 0;
+let k = 1;
+
+$: y = (x*x+k) * (x*x+k) - k;
+```
+
+And want to abstract perhaps it's natural to think of there as being a function `f(x)` with `k` as a global variable:
```

Comment:
How about this? If I understood correctly:
```suggestion
And you want to abstract it. Perhaps it's natural to think of there being a function `f(x)` with `k` as a global variable:
```
But overall, nice explanation btw!